migrations/20210618-fix-function-search-path.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var Sequelize = require('sequelize');
+
+/**
+ * Actions summary:
+ *
+ * alter functions to add public search path
+ *
+ **/
+
+var info = {
+    "revision": 26,
+    "name": "fix-function-search-path",
+    "created": "2021-06-18T11:11:39.431Z",
+    "comment": ""
+};
+
+var migrationCommands = function(transaction) {
+    return [];
+};
+var rollbackCommands = function(transaction) {
+    return [];
+};
+
+module.exports = {
+    pos: 0,
+    useTransaction: true,
+    execute: function(queryInterface, Sequelize, _commands)
+    {
+        var index = this.pos;
+        function run(transaction) {
+            const commands = _commands(transaction);
+            return new Promise(function(resolve, reject) {
+                function next() {
+                    if (index < commands.length)
+                    {
+                        let command = commands[index];
+                        console.log("[#"+index+"] execute: " + command.fn);
+                        index++;
+                        queryInterface[command.fn].apply(queryInterface, command.params).then(next, reject);
+                    }
+                    else
+                        resolve();
+                }
+                next();
+            });
+        }
+        if (this.useTransaction) {
+            return queryInterface.sequelize.transaction(run);
+        } else {
+            return run(null);
+        }
+    },
```

Comment:
omit this?

Reply:
its default in any, so idk

Reply:
it wont break anything and make the file easier to read eg like https://github.com/aeternity/tipping-community-backend/blob/main/migrations/20201005-postfix-tip-id.js